### ISSUE TITLE
GCpedia error -- urgent

### ISSUE THREAD

**Issue by Menrika** [gctools-outilsgc/gcconnex#2551]
## In the last few days, we noticed on all our pages with a Google Analytics tag a bug preventing any attempt to update:

The requested URL was rejected. Please consult with your administrator. Your support ID is: 8952924826835424499 [Go Back]

https://www.gcpedia.gc.ca/wiki/Client_Relations_Portal_-_Newsletter

## Details on issue or enhancement

Started April 18th

https://support.gccollab.ca/a/tickets/28910

https://support.gccollab.ca/a/tickets/29283 

We believe this ticket has escalated to higher levels 

all GCPEDIA pages with Google Analytics trackers returned an error code preventing them from being updated. With everyone interacting with those pages.

Will ask them for the rest of the pages

Possible cause : Windows (Quality Updates - KB5025305 (OS Build 22621.1635) released on April 25, 2023

## For the development team
- [ ] Issue user story documented
- [ ] UX input received
- [ ] Design completed
- [ ] Design validated by business team / UX
- [ ] Code review completed by peer
- [ ] Issue closing comment references any duplicate or connected issues or pull requests
- [ ] Issue closed

**Comment by piet0024** [comment]
This is most likely an issue with the firewall. I will add this to the sprint to investigate.
To clarify is this an issue with the page owner trying to add their own Google Analytics to their page?

**Comment by Phanoix** [comment]
Yes, that would be their own GA tags / script. I passed it on already with the ID.

**Comment by Menrika** [comment]
First page fixed, they will be testing the other pages later
